structured-query/documentation/2021_01_29StructeredQueriesDocumentation(Draft).md

```diff
@@ -0,0 +1,132 @@
+# Structured Queries
+
+CODEX front-end allows users to create requests based on inclusion and exclusion criteria. The different inclusion and exclusion criteria are conjuncted with the "AND" and "OR" operators respectively. Resulting in a conjunctive normal form (CNF) for inclusion and disjunctive normal form (DNF) for the exclusion  criteria.
+
+The front-end created queries need to be transmitted to different back-end services which translate the CNF and DNF to queries. As common data exchange format the so called *Structured Queries* are defined.
```

Comment:
[...] which translates the Structured Query format into other query formats such as FHIR Search or CQL.

Reply:
Accepted.